src/pyscaffold/cli.py

```diff
@@ -102,17 +102,30 @@ def add_default_args(parser: argparse.ArgumentParser):
         help="update an existing project by replacing the most important files"
         " like setup.py etc. Use additionally --force to replace all scaffold files.",
     )
+    parser.add_argument(
+        "-f",
+        "--force",
+        dest="force",
+        action="store_true",
+        required=False,
+        help="force overwriting an existing directory",
+    )
 
     # The following are basically for the CLI options, so having a default value is OK.
     parser.add_argument(
-        "-V", "--version", action="version", version=f"PyScaffold {pyscaffold_version}"
+        "-V",
+        "--version",
+        action="version",
+        noprompt=True,
```

Comment:
Would `prompt=False` be better than `noprompt=True`?

Reply:
Absolutely, totally agree! What was I thinking? Normally I try to avoid double negations :-)